From bdd9b9807a89eb8e4789a3279253f6f845f90e9b Mon Sep 17 00:00:00 2001
From: Michal Schmidt <mschmidt@redhat.com>
Date: Thu, 15 Dec 2011 21:32:50 +0100
Subject: [PATCH] tmpfiles: rename a couple of functions

remove_item -> remove_item_instance
remove_item_glob -> remove_item
(cherry picked from commit a08961233bea3bfad7776e04482d07012216b134)
---
 src/tmpfiles.c |    8 ++++----
 1 files changed, 4 insertions(+), 4 deletions(-)

diff --git a/src/tmpfiles.c b/src/tmpfiles.c
index 6171140..4308f7f 100644
--- a/src/tmpfiles.c
+++ b/src/tmpfiles.c
@@ -563,7 +563,7 @@ finish:
         return r;
 }
 
-static int remove_item(Item *i, const char *instance) {
+static int remove_item_instance(Item *i, const char *instance) {
         int r;
 
         assert(i);
@@ -599,7 +599,7 @@ static int remove_item(Item *i, const char *instance) {
         return 0;
 }
 
-static int remove_item_glob(Item *i) {
+static int remove_item(Item *i) {
         assert(i);
 
         switch (i->type) {
@@ -633,7 +633,7 @@ static int remove_item_glob(Item *i) {
                 }
 
                 STRV_FOREACH(fn, g.gl_pathv)
-                        if ((k = remove_item(i, *fn)) < 0)
+                        if ((k = remove_item_instance(i, *fn)) < 0)
                                 r = k;
 
                 globfree(&g);
@@ -650,7 +650,7 @@ static int process_item(Item *i) {
         assert(i);
 
         r = arg_create ? create_item(i) : 0;
-        q = arg_remove ? remove_item_glob(i) : 0;
+        q = arg_remove ? remove_item(i) : 0;
         p = arg_clean ? clean_item(i) : 0;
 
         if (r < 0)
